src/pages/NewsDetails/components/NewsRenderer.tsx

```diff
@@ -58,9 +59,28 @@ const NewsRenderer = ({ data, preview = false }: NewsRendererProps) => {
               </Button>
             </HavePermission>
           )}
-          <Stack alignItems='center' direction='row' justifyContent='space-between'>
-            <Typography variant='body2'>Publisert: {formatDate(parseISO(data.created_at), { time: false })}</Typography>
-            <ShareButton shareId={data.id} shareType='news' title={data.title} />
+          <Stack alignItems='center' direction={{ xs: 'column', md: 'row' }} justifyContent='space-between'>
+            <div>
+              {data.creator && (
+                <Stack direction='column'>
+                  <Typography textAlign='center' variant='body2'>
+                    Forfatter:
+                  </Typography>
+                  <Button component={Link} to={`${URLS.profile}${data.creator.user_id}/`}>
+                    <Avatar user={data.creator} />
+                    <Box sx={{ mx: 2 }}>
+                      <Typography variant='body1'>
+                        {data.creator.first_name} {data.creator.last_name}
+                      </Typography>
+                    </Box>
+                  </Button>
+                </Stack>
+              )}
+            </div>
+            <Stack alignItems='center' direction='row' gap={3}>
+              <Typography variant='body2'>Publisert: {formatDate(parseISO(data.created_at), { time: false })}</Typography>
+              <ShareButton shareId={data.id} shareType='news' title={data.title} />
+            </Stack>
           </Stack>
```

Comment:
Kan jeg foreslå en slik løsning med bare en link til forfatter:
```suggestion
          <Stack alignItems='center' direction='row' justifyContent='space-between'>
            <Stack>
              <Typography variant='body2'>Publisert: {formatDate(parseISO(data.created_at), { time: false })}</Typography>
              {data.creator && (<Typography variant='body2'>Forfatter: <Link to={`${URLS.profile}${data.creator.user_id}/`}>{data.creator.first_name} {data.creator.last_name}</Link></Typography>)}
            </Stack>
            <ShareButton shareId={data.id} shareType='news' title={data.title} />
          </Stack>
```